sparse/numba_backend/tests/test_einsum.py

```diff
@@ -82,6 +82,15 @@
 ]
 
 
+@pytest.mark.parametrize("symbols", sparse.numba_backend._common._EINSUM_SYMBOLS_SET)
```

Comment:
```suggestion
@pytest.mark.parametrize("symbols", sparse.numba_backend._common._EINSUM_SYMBOLS)
```

This should get rid of the test failures

Reply:
For context, `set` is unordered in Python (which can be confusing as `dict` IS ordered). So each runner ends up with a different order of the same tests, which apparently trips up `pytest-xdist` (the framework we use to parallelize tests).